sympy/integrals/intpoly.py

```diff
@@ -65,6 +65,8 @@ def polytope_integrate(poly, expr=None, **kwargs):
     if clockwise is True:
         if isinstance(poly, Polygon):
             poly = point_sort(poly)
+            if not isinstance(poly, Polygon):
+                poly = Polygon(*poly)
```

Comment:
I think it makes sense to do this here. You might want to use `evaluate=False` so no present (or future) Polygon instantiation will change the ordering.